nequip/data/AtomicData.py

```diff
@@ -249,6 +250,44 @@ def from_ase(cls, atoms, r_max, **kwargs):
             **add_fields,
         )
 
+    def to_ase(self) -> Union[List[ase.Atoms], ase.Atoms]:
+        """Build a (list of) ``ase.Atoms`` object(s) from an ``AtomicData`` object.
+
+        For each unique batch number associated with AtomicDataDict.BATCH_KEY,
+        an ``ase.Atoms`` object is created. If AtomicDataDict.BATCH_KEY does not
+        exist in self, a single ``ase.Atoms`` object is created.
+
+        Returns:
+            A list of ``ase.Atoms`` objects if AtomicDataDict.BATCH_KEY is in self
+            and is not None. Otherwise, a single ``ase.Atoms`` object is returned.
+        """
+        positions = self.pos
+        atomic_nums = self.atomic_numbers
+        pbc = self.pbc if AtomicDataDict.PBC_KEY in self else None
+        cell = self.cell if AtomicDataDict.CELL_KEY in self else None
+        batch = self.batch if AtomicDataDict.BATCH_KEY in self else None
```

Comment:
A nice Python-ism for this is `getattr`, which accepts a default value to be returned if the attribute with the given name isn't found:
```
        pbc = getattr(self, AtomicDataDict.PBC_KEY, None)
                            # ^ obj           ^ key                           ^ default
```
and so on